### ISSUE TITLE
Is the `form-data` dependency needed?

### ISSUE THREAD

**Issue by vsund** [ntzwrk/blockstack.ts#13]
In [`src/network.js`](https://github.com/blockstack/blockstack.js/blob/286df82401c5be8d89e352054e2192e0f600ea98/src/network.js#L3) there's an import of [`form-data`]().
`tsc` complains about incompatible types at its [only usage](https://github.com/blockstack/blockstack.js/blob/286df82401c5be8d89e352054e2192e0f600ea98/src/network.js#L666).

When I remove the dependency `tsc` says it's fine. Is this dependency purposely in there?

**Comment by kantai** [comment]
Yeah -- it's required for submitting that request as a form submission (i.e., `multipart/form-data`) -- the FormData object is present in many browsers, but its not in NodeJS, so we needed to add it.
